codespell_lib/data/dictionary.txt

```diff
@@ -29104,6 +29126,7 @@ unidentifiedly->unidentified
 unidimensionnal->unidimensional
 unifiy->unify
 uniformely->uniformly
+uniformy->uniformly
```

Comment:
Suggest uniform too.

Reply:
added.

Reply:
From a look at grep.app:
https://grep.app/search?current=3&q=uniformy&words=true

When they type uniformy they generally seem to mean uniformly, so we should put that correction first.

Reply:
makes sense, changed

Reply:
Ta